noxfile.py

```diff
@@ -139,18 +133,8 @@ def _check(env: Dict[str, str]) -> None:
         env = env or os.environ
         for feature_set in _get_feature_sets():
             try:
-                _run(
-                    session,
-                    "cargo",
-                    "check",
-                    *feature_set,
-                    "--all-targets",
-                    "--workspace",
-                    # linting pyo3-ffi-check requires docs to have been built or
-                    # the macros will error; doesn't seem worth it on CI
-                    "--exclude=pyo3-ffi-check",
-                    external=True,
-                    env=env,
+                _run_cargo(
+                    session, *feature_set, "--all-targets", "--workspace", env=env
```

Comment:
Missing "check"

Reply:
Thanks, the classic editing-code-last-thing-at-night mistake - at least we have CI to check me! 😄